src/litserve/loops/continuous_batching_loop.py

```diff
@@ -197,7 +197,8 @@ async def run(
                 # Process one step for all active sequences
                 responses = await self.step(prev_outputs, lit_api, lit_spec)
                 if len(responses) == 0:
-                    raise HTTPException(500, "No responses from step()")
+                    logger.warning("No responses from step() but has_active_requests() is true")
```

Comment:
could we get a test for this?

Reply:
logger warning or regular warning?

Reply:
yeah, i will add in a separate PR. the whole continous batching thing needs test.